src/domain/resource/utils.js

```diff
@@ -190,8 +190,8 @@ export const getFullCalendarBusinessHours = (resource, date = null) => {
       const dayNumber = Number(moment(item.date).format('E'));
       businessHours.push({
         daysOfWeek: [dayNumber < 7 ? dayNumber : 0],
-        startTime: item.opens ? moment(item.opens).format('HH:mm') : '00:00',
-        endTime: item.closes ? moment(item.closes).format('HH:mm') : '00:00',
+        startTime: item.opens ? moment(item.opens).tz('Europe/Helsinki').format('HH:mm') : '00:00',
```

Comment:
As i debug, seems like timezone is not injected globally, we should not fix that like this